src/drop-down-menu.jsx

```diff
@@ -24,6 +24,7 @@ let DropDownMenu = React.createClass({
     displayMember: React.PropTypes.string,
     valueMember: React.PropTypes.string,
     autoWidth: React.PropTypes.bool,
+    disabled: React.PropTypes.bool,
```

Comment:
can you add the default value false in getDefaultProps so that it's straightforward?